packages/jest-cli/src/cli/index.js

```diff
@@ -142,7 +142,11 @@ const getProjectListFromCLIArgs = (argv, project: ?Path) => {
   }
 
   if (!projects.length) {
-    projects.push(process.cwd());
+    if (process.platform === 'win32') {
+      projects.push(process.binding('fs').realpath(process.cwd()));
```

Comment:
why not `fs.realpathSync()`? `process.binding` seems overkill

Reply:
fs.realpathSync() does not correct the path.

They differ:
```
fs.realpathSync(process.cwd()):
C:\Users\pdanis\desktop\\projects\my-js-learning\other\temp

process.binding('fs').realpath(process.cwd()):
C:\Users\pdanis\Desktop\Projects\my-js-learning\other\temp

Reply:
Huh, that's weird

Reply:
If I understand it correctly `fs.realpathSync()` uses `JS` fs and `process.binding('fs').realpath()` is using  `GetFinalPathNameByHandle()` on Windows.

There is a PR, which should expose `process.binding('fs').realpath()` as `fs.realpathSync.native()`.
https://github.com/nodejs/node/pull/15776